### DISCUSSION TITLE
Content Assist

### DISCUSSION THREAD

**Discussion by brianrocket13** [codefori/vscode-ibmi#1753] [Q&A] [answered]
How do I get rid of content assist? I find it very annoying. No matter what I do including putting in a blank it pops up.  It does not work with nested data structures. Sorry, I'm used to RDi content assist which I really like. This is for RPG. I'm on Code for IBM i v2.6.2 and VS Code 1.84.2. Thanks, Brian

**Comment by worksofliam** [comment, marked as answer]
There is a VS Code setting you can use to turn off the content sssist trigger.

Search for this and set it to false (turn it off): suggestOnTriggerCharacters

**Reply by brianrocket13** [reply]
Thank you so much!  It's off now and am enjoying VS Code for IBM i much more now
